logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -37,3 +47,61 @@ def format_stackdriver_json(record, message):
     }
 
     return json.dumps(payload)
+
+
+def get_trace_id_from_flask():
+    """Get trace_id from flask request headers.
+
+    :rtype: str
+    :return: Trace_id in HTTP request headers.
+    """
+    if not flask or not flask.request:
+        return None
+
+    header = flask.request.headers.get(_FLASK_TRACE_HEADER)
+
+    if not header:
+        return None
+
+    trace_id = header.split('/')[0]
```

Comment:
The format for this header is `<trace-id>[/<span-id>][;o=<trace-options>]`, if the header presents, the trace_id should be valid in this format. And even there is no `/` in the header, the trace_id will just be the whole string.